packages/ipfs-grpc-server/test/utils/channel.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+const pushable = require('it-pushable')
+
+class MessageChannel {
+  constructor () {
+    this.source = pushable()
+    this.sink = pushable()
+
+    this.clientSink = pushable()
+  }
+
+  sendMetadata (metadata) {
+    this.metadata = metadata
+  }
+
+  sendMessage (message) {
+    setTimeout(() => {
```

Comment:
why the setTimeout ?

Reply:
To disconnect the ends of the channel in the tests, helps simulate the other end of the channel being over a network connection.